src/ecalc/libraries/libecalc/common/tests/input/validation/snapshots/test_validation_json_schemas/test_json_schema_changed/schemas.json

```diff
@@ -32,20 +70,32 @@
                 },
                 "TIME_SERIES": {
                     "$ref": "https://test.ecalc.equinor.com/api/v1/schema-validation/time-series.json#properties/TIME_SERIES",
-                    "description": "Defines the inputs for time dependent variables, or \u201creservoir variables\u201d.\n\nhttps://test.ecalc.equinor.com/docs/docs/modelling/keywords/TIME_SERIES",
+                    "description": "Defines the inputs for time dependent variables, or 'reservoir variables'.\n\nhttps://test.ecalc.equinor.com/docs/docs/modelling/keywords/TIME_SERIES",
                     "title": "TIME_SERIES"
                 },
                 "VARIABLES": {
-                    "$ref": "https://test.ecalc.equinor.com/api/v1/schema-validation/definitions.json#definitions/VARIABLES"
+                    "additionalProperties": {
```

Comment:
I assume this was missing in the old json schema, but should have been there?

Reply:
The difference is that it was referenced before, now we are adding it directly or in definitions. So we are going to end up with a single schema "file" instead of several.